griptape/structures/structure.py

```diff
@@ -37,8 +32,8 @@ class ConversationMemoryStrategy(Enum):
         kw_only=True,
         metadata={"serializable": True},
     )
-    conversation_memory_strategy: ConversationMemoryStrategy = field(
-        default=ConversationMemoryStrategy.PER_STRUCTURE, kw_only=True, metadata={"serializable": True}
+    conversation_memory_strategy: Literal["per_structure", "per_task"] = field(
+        default="per_structure", kw_only=True, metadata={"serializable": True}
```

Comment:
do we need a validator that checks the value is correct?

Reply:
I'd rather rely on the type checker for this.